### ISSUE TITLE
Implement API get group detail (chatting message data) by gr code/id

### ISSUE THREAD

**Issue by secretdeveloperisme** [tienphuckx/anony-box-backbone#13]

**Comment by tienphuckx** [comment]
It seem like we need to handle check the remain time before load the gr detail @secretdeveloperisme

**Comment by secretdeveloperisme** [comment]
> It seem like we need to handle check the remain time before load the gr detail [@secretdeveloperisme](https://github.com/secretdeveloperisme)

Yes, it is necessary to check expired time before responding group detail from server, but I am thinking about deleting expired groups' behavior, should we need to delete them immediately or at the end of the day
